projects/hello_world/README.md

```diff
@@ -0,0 +1,16 @@
+<!--
+    General guidelines
+    These are just guidelines, not strict rules - document however seems best.
+    A README for a firmware-only project (e.g. Babydriver, MU, bootloader, CAN explorer) should answer the following questions:
+        - What is it?
+        - What problem does it solve?
+        - How do I use it? (with usage examples / example commands, etc)
+        - How does it work? (architectural overview)
+    A README for a board project (powering a hardware board, e.g. power distribution, centre console, charger, BMS carrier) should answer the following questions:
+        - What is the purpose of the board?
+        - What are all the things that the firmware needs to do?
+        - How does it fit into the overall system?
+        - How does it work? (architectural overview, e.g. what each module's purpose is or how data flows through the firmware)
+-->
+# hello_world
```

Comment:
hello_world should not be included with the firmware 102 homework

We normally want to keep each project on separate branches (or have multiple branches for a single project if its a big project) so that we don't have gigantic pull requests.